lib/init/config-initializer.js

```diff
@@ -445,110 +448,115 @@ function askInstallModules(modules, packageJsonExists) {
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "moduleType",
             message: "What type of modules does your project use?",
-            default: "esm",
+            initial: 0,
             choices: [
-                { name: "JavaScript modules (import/export)", value: "esm" },
-                { name: "CommonJS (require/exports)", value: "commonjs" },
-                { name: "None of these", value: "none" }
+                { message: "JavaScript modules (import/export)", name: "esm" },
+                { message: "CommonJS (require/exports)", name: "commonjs" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "framework",
             message: "Which framework does your project use?",
-            default: "react",
+            initial: 0,
             choices: [
-                { name: "React", value: "react" },
-                { name: "Vue.js", value: "vue" },
-                { name: "None of these", value: "none" }
+                { message: "React", name: "react" },
+                { message: "Vue.js", name: "vue" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
             type: "confirm",
             name: "typescript",
             message: "Does your project use TypeScript?",
-            default: false
+            initial: () => false
         },
         {
-            type: "checkbox",
+            type: "multiselect",
             name: "env",
             message: "Where does your code run?",
-            default: ["browser"],
+            initial: 0,
             choices: [
-                { name: "Browser", value: "browser" },
-                { name: "Node", value: "node" }
+                { message: "Browser", name: "browser" },
+                { message: "Node", name: "node" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "source",
             message: "How would you like to define a style for your project?",
-            default: "guide",
             choices: [
-                { name: "Use a popular style guide", value: "guide" },
-                { name: "Answer questions about your style", value: "prompt" },
-                { name: "Inspect your JavaScript file(s)", value: "auto" }
+                { message: "Use a popular style guide", name: "guide" },
+                { message: "Answer questions about your style", name: "prompt" },
+                { message: "Inspect your JavaScript file(s)", name: "auto" }
             ],
-            when(answers) {
-                return answers.purpose === "style";
+            skip() {
+                return this.state.answers.purpose !== "style";
+            },
```

Comment:
The `skip` method is the reverse of `when` so the conditional statement is reversed.